Source/WebKit/UIProcess/Extensions/WebExtensionContext.h

```diff
@@ -518,6 +519,13 @@ class WebExtensionContext : public API::ObjectImpl<API::Object::Type::WebExtensi
     // Storage
     void setSessionStorageAllowedInContentScripts(bool);
     bool isSessionStorageAllowedInContentScripts() const { return m_isSessionStorageAllowedInContentScripts; }
+    double quoataForStorageType(WebExtensionStorageType);
+
+    RetainPtr<_WKWebExtensionStorageSQLiteStore> localStorageStore();
+    RetainPtr<_WKWebExtensionStorageSQLiteStore> sessionStorageStore();
+    RetainPtr<_WKWebExtensionStorageSQLiteStore> syncStorageStore();
+    RetainPtr<_WKWebExtensionStorageSQLiteStore> storageForType(WebExtensionStorageType);
```

Comment:
These should just return `_WKWebExtensionStorageSQLiteStore *`. Since we use ARC, we don't need to return `RetainPtr`. This applies to the other storage getters like `registeredContentScriptsStore()` too.